cognite_toolkit/_cdf_tk/load/_resource_loaders.py

```diff
@@ -1648,28 +1648,39 @@ def load_resource(
 
         for resource in resources:
             try:
-                resource["config"] = yaml.dump(resource.get("config", ""), indent=4)
+                if resource.get("config", None):
+                    resource["config"] = yaml.dump(resource.get("config"), indent=4)
             except Exception:
                 print(
                     f"[yellow]WARNING:[/] configuration for {resource.get('external_id')} could not be parsed as valid YAML, which is the recommended format.\n"
                 )
-                resource["config"] = resource.get("config", "")
+                resource["config"] = resource.get("config", None)
 
         if len(resources) == 1:
             return ExtractionPipelineConfigWrite.load(resources[0])
         else:
             return ExtractionPipelineConfigWriteList.load(resources)
 
-    def create(self, items: ExtractionPipelineConfigWriteList) -> ExtractionPipelineConfigList:
-        created = ExtractionPipelineConfigList([])
+    def _upsert(self, items: ExtractionPipelineConfigWriteList) -> ExtractionPipelineConfigList:
+        updated = ExtractionPipelineConfigList([])
         for item in items:
-            item_created = self.client.extraction_pipelines.config.create(item)
-            created.append(item_created)
-        return created
+            if not item.external_id:
+                raise ValueError("ExtractionPipelineConfig must have external_id set.")
```

Comment:
This is a user fault? Then, you need a `ToolkitError` as a regular `ValueError` will cause sentry to fire